src/player/PlayerQuickItem.cpp

```diff
@@ -104,19 +91,38 @@ bool PlayerRenderer::init()
   DwmEnableMMCSS(TRUE);
 #endif
 
-  mpv_opengl_cb_set_update_callback(m_mpvGL, on_update, (void *)this);
+  static mpv_opengl_init_params opengl_init_params = {
+      get_proc_address, // .get_proc_address
```

Comment:
Whats this comment for, seems like it's not needed?

Reply:
Indicates which member, since C++ doesn't have C-style designated initializers

Reply:
Yeah, the windows compiler won't allow to use C++11 initializers, so those are the struct field names.

Reply:
Isn't it obvious that `get_proc_address` == `.get_proc_address`, maybe I'm missing something?

Reply:
Well it turns out that our function has the same name as the struct property, but those could be different names. as we can't specify which member gets filled, and that it's only determined by the order or the params, so could as well be `foo`barlue going into `.bar` member.
We added that as well for all the other struct initializers.